bundle/src/Server.ts

```diff
@@ -56,7 +58,32 @@ async function main() {
 		// },
 	} as any);
 
+	//Sentry
+	if (Config.get().sentry.enabled) {
+		console.log(
+			`[Bundle] ${yellow("You are using Sentry! This may slightly impact performance on large loads!")}`
+		);
+		Sentry.init({
```

Comment:
do not initalize sentry in the bundle and do it in the api instead

Reply:
Wouldnt that make it not get CDN paths etc?
The entire point is to have error monitoring for everything in one place.
Essentially: I want to cover as many paths as possible.